core/templates/dev/head/pages/exploration_editor/settings_tab/settings_tab.html

```diff
@@ -336,6 +336,57 @@
     </div>
   </md-card>
 
+  <md-card class="oppia-editor-card">
+    <h3>Feedback/Suggestion Email Preferences</h3>
+    <div class="row">
+      <div class="row">
+        <div class="col-lg-12 col-md-12 col-sm-12">
+          <div role="form" class="form-horizontal">
+            <label for="FeedbackNotifications" class="col-lg-3 col-md-3 col-sm-3">
+              Feedback emails
+            </label>
+            <span class="col-lg-2 col-md-2 col-sm-2">
+              <button type="button" class="btn btn-default protractor-test-enable-fallbacks"
+              ng-click="muteFeedbackNotifications()" ng-if="!userExplorationEmailsHandler.areFeedbackNotificationsMuted()">
```

Comment:
Our convention is to align the left edge of 'ng-click' with left edge of 'type'. Ditto below.